test/func_network.py

```diff
@@ -85,9 +86,12 @@ def network_pktdriver_mtcp(self, driver):
 $_tapdev = "tap0"
 """, timeout=30)
 
-    p.join()
-    sleep(5)
-    p.close()
+    p.join(timeout=45)
+    if p.is_alive():
+        stderr.write("Timeout on join() for little_webserver - killing ")
+        stderr.flush()
+        p.kill()
+        sleep(5)    # wait for the process to die before attempting teardown
```

Comment:
Ah, ignore prev comment. :)
So where is `p.close()`?

Reply:
Apparently it's only a recent addition to python multiprocessing and not required. It was added for those people who wanted to explicitly release python resources rather than rely upon the automatic garbage collection. See https://stackoverflow.com/questions/58866837/python3-multiprocessing-terminate-vs-kill-vs-close.

That said I think I want to change the sleep(5) into a p.join(5).

Reply:
OK, as you wish, but to me `close()`
looks more natural then the lack of it.

> That said I think I want to change the sleep(5) into a p.join(5).

For what?
Even the successful join still left the
TAP problem unsolved, so I guess we
need both join and sleep.

Reply:
So I updated the commit. I now join(timeout=5) after the kill(), then sleep() in the teardown function.